highlight.go

```diff
@@ -57,11 +53,12 @@ type HTMLConfig struct {
 	HTMLAttrName  string
 	HTMLAttrValue string
 	Decimal       string
+	Whitespace    string
 }
 
 type HTMLPrinter HTMLConfig
 
-func (c HTMLConfig) class(kind Kind) string {
+func (c HTMLConfig) Class(kind Kind) string {
```

Comment:
Now that this method is exported, can you add a short comment documenting what it does? This is something `golint` would suggest. E.g., similar to how I've done in https://github.com/sourcegraph/syntaxhighlight/pull/17/files, it can be very short.